### PR TITLE
New Plugin: Easee

### PR DESCRIPTION
**Pull request by mzanetti**
nymea-plugins pull request checklist:

- [x] Make sure the pull request's title is of format "Plugin name: Add support for xyz" or "New plugin: Plugin name"

- [x] Did you test the changes on hardware, if not (e.g. absence of required hardware), please mention a person to confirm it has been tested.

- [x] Did you update the plugin's README.md accordingly?

- [x] Did you update translations (`cd builddir && make lupdate`)?

### CODE REVIEW

File: easee/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+This integration allows nymea to control Easee wallboxes.
+
+## Supported things
+
+All models of Easee wallboxes are supported.
+
+## Requirements and setup
+
+It is required to create an account at ease and connect the wallbox to that account. Generally, setting up the Wallbox using the Easee app will provide everything necessary.
```
Comment (t-mon) [inline easee/README.md:11]: account at ease`e`

File: easee/integrationplugineasee.cpp [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+* For any further details and any questions please contact us under
+* contact@nymea.io or see our FAQ/Licensing Information on
+* https://nymea.io/license/faq
+*
+* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+
+#include "integrationplugineasee.h"
+#include "plugininfo.h"
+#include "plugintimer.h"
```
Comment (t-mon) [inline easee/integrationplugineasee.cpp:34]: <plugintimer.h>

File: easee/integrationplugineasee.cpp [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+                    break;
+                }
+            }
+        });
+    }
+}
+
+void IntegrationPluginEasee::thingRemoved(Thing *thing)
+{
+    Q_UNUSED(thing)
```
Comment (t-mon) [inline easee/integrationplugineasee.cpp:274]: you should also remove the entire thing id group in the plugin storage

Comment (t-mon) [reply easee/integrationplugineasee.cpp:274]:
```
pluginStorage()->beginGroup(thing->id().toString());
pluginStorage()->remove("");
pluginStorage()->endGroup();
```

**Review by t-mon — CHANGES_REQUESTED** [review]

File: easee/integrationplugineasee.cpp (hunk 3 above)
Comment (mzanetti) [reply easee/integrationplugineasee.cpp]: good catch. fixed

File: easee/integrationplugineasee.cpp (hunk 2 above)
Comment (mzanetti) [reply easee/integrationplugineasee.cpp]: done

File: easee/README.md (hunk 1 above)
Comment (mzanetti) [reply easee/README.md]: done

**Review by t-mon — APPROVED** [review]

**Pull request merged**